Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Reflection;
```

Comment:
It is needed for the command line parser. It reads out assembly information to display the name, copyright information etc. If you don't add this, the command line parser goes into an infinite loop actually (found that out because I didn't have it at first)